src/components/navbar/NavItem.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { Button } from "antd";
+import "./NavBar.css";
+import {NavItemProps} from "../../interfaces/Navbar.ts";
+const NavItem: React.FC<NavItemProps> = ({ to, label, Icon, isActive }) => {
+    const [isHovered, setIsHovered] = useState(false);
+
+    return (
+        <Link to={to}>
+            <Button
+                className={`text-sm font-bold flex items-center space-x-2 px-4 py-2 rounded-lg transition-colors duration-200 ${
```

Comment:
```suggestion
                className={`h-7 text-sm font-bold flex items-center space-x-2 px-4 py-2 rounded-lg transition-colors duration-200 ${
```
add h-7 to adjust NavItem's size